docs/source/en/model_doc/gptj.mdx

```diff
@@ -105,6 +105,30 @@ model.
 >>> gen_text = tokenizer.batch_decode(gen_tokens)[0]
 ```
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with GPT-J. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-generation"/>
+- A blog on how to [Deploy GPT-J 6B for inference using Hugging Face Transformers and Amazon SageMaker](https://huggingface.co/blog/gptj-sagemaker)
+- A blog on how to [Accelerate GPT-J inference with DeepSpeed-Inference on GPUs](https://www.philschmid.de/gptj-deepspeed-inference)
+- A blog post introducing GPT-J [GPT-J-6B: 6B JAX-Based Transformer](https://arankomatsuzaki.wordpress.com/2021/06/04/gpt-j/) 🌎
+- A notebook for [GPT-J-6B Inference Demo](https://colab.research.google.com/github/kingoflolz/mesh-transformer-jax/blob/master/colab_demo.ipynb)
+- [Causal language modeling](https://huggingface.co/course/en/chapter7/6?fw=pt#training-a-causal-language-model-from-scratch) chapter of the 🤗 Hugging Face Course.
+- [`GPT2LMHeadModel`] is supported by this [causal language modeling example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#gpt-2gpt-and-causal-language-modeling), [text generation example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-generation), and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/language_modeling.ipynb).
```

Comment:
Replace GPT2 with GPTJ, same for the TF and Flax implementations

```suggestion
- [`GPTJCausalLM`] is supported by this [causal language modeling example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#gpt-2gpt-and-causal-language-modeling), [text generation example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-generation), and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/language_modeling.ipynb).
```

Reply:
I have updated the labels. However, looking at the links, they mention GPT-2 but GPT-J is not mentioned as being supported. For example the script mentioned here: https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#gpt-2gpt-and-causal-language-modeling. 

My questions are:

(1) I am a bit confused about what these links are. Are these just scripts that make the modeling process easier? Clarification on what these are would be appreciated.

(2) Are there any scripts present which support GPT-J? I had a look but couldn't find anything.

Reply:
1. These are example scripts for how to finetune a model for a certain task if you prefer to just run a script instead of a notebook. 
2. The script above for causal language modeling (CLM) should support GPT-J and all models in general with a CLM pretraining objective (so anything in the GPT family).